maquetado/scss/partials/_tech.scss

```diff
@@ -15,12 +16,9 @@ $border-radius: 2%;
   padding: 30px;
   position: relative;
   width: 200px;
-}
-
-.tech-desc {
-  text-align: center;
-
+  
   &::after {
+    @include transition(all 0.5s ease);
```

Comment:
Try not to apply transitions to all the CSS properties, it may result in a performance bottleneck. Take a look at [this  article](https://blog.gyrosco.pe/smooth-css-animations-7d8ffc2c1d29), it explains some good tips for web animations/transitions. 🙂